docs/get-started/VeloxIceberg.md

```diff
@@ -91,6 +91,11 @@ Not support avro format.
 ## SQL
 Only support SELECT.
 
+## Schema evolution
+PartialOffload
```

Comment:
Maybe make it a subtitle. Otherwise, it will be combined with the following words into one line.